### PR TITLE
Acl export acls with proper tags encoding

### PR DESCRIPTION
**Pull request by DaanHoogland**
### Description

This PR...

Fixes: #8256

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan ui

**Review by OlegChuev — CHANGES_REQUESTED** [review]

File: ui/src/views/network/AclListRulesTab.vue [hunk 1]
```diff
@@ -334,7 +334,17 @@ export default {
             result += columnDelimiter
           }
 
-          result += typeof item[key] === 'string' && item[key].includes(columnDelimiter) ? `"${item[key]}"` : item[key]
+          if (key === 'tags') {
+            var tags = '"'
+            if (item[key].length > 0) {
+              item[key].forEach(tag => {
+                tags += '(' + tag.key + ',' + tag.value + ')'
```
[the remark this answers is not in the document]
Comment (DaanHoogland) [reply ui/src/views/network/AclListRulesTab.vue:341]: ok, should we include double quotes around the keys and values or add escapes. This si really new functionality so we can make choices here.
Honestly, I think this si good as a first implementation and we should think about adding options like
- export as json
- do escaping
- use sinlge or double quotes

cc @shwstppr

[the remark this answers is not in the document]
Comment (shwstppr) [reply ui/src/views/network/AclListRulesTab.vue:341]: I like the idea of json export

**Comment by shwstppr** [comment]
@DaanHoogland I guess this needs more changes?

**Comment by DaanHoogland** [comment]
> @DaanHoogland I guess this needs more changes?

I think it can be merged as is, any extra would be an enhancement on top. We can create new issues for those.

**Comment by shwstppr** [comment]
@blueorangutan ui

**Comment by shwstppr** [comment]
@blueorangutan ui

**Review by harikrishna-patnala — APPROVED** [review]
code LGTM, wrt to tags issue

**Pull request merged**